fiftyone/core/collections.py

```diff
@@ -7140,13 +7267,15 @@ def _parse_field_name(
     omit_terminal_lists,
     allow_missing,
 ):
-    unwind_list_fields = set()
+    unwind_list_fields = []
+    explicit_unwinds = set()
```

Comment:
@benjaminpkane the change you made here is breaking [this unit test](https://github.com/voxel51/fiftyone/blob/866b17a436a3399abc379b1bea4ad4fd56bf67ea/tests/unittests/view_tests.py#L1303-L1310).

What were you trying to achieve? This unwinding code is tricky; I can take a shot at supporting what you need

Reply:
I want to support `ListField(FrameSupportField)` fields in the App sidebar filters. This requires computing the bounds on a nested list.

```py
import fiftyone.zoo as foz


dataset = foz.load_zoo_dataset("quickstart")
dataset.add_sample_field("support_list", fo.ListField, subfield=fo.FrameSupportField)
sample = dataset.first()
sample["support_list"] = [[1, 1], [2, 2]]

dataset.bounds("support_list")
```

Reply:
@benjaminpkane ah I see, I added support for this via the syntax: `dataset.bounds("support_list[][]")`

Reply:
Okay even better just `dataset.bounds("support_list")` now works again